### DISCUSSION TITLE
Refer to JEEWMS to develop more functions

### DISCUSSION THREAD

**Discussion by Jengro777** [GreaterWMS/GreaterWMS#338] [General]
https://gitee.com/erzhongxmu/JEEWMS?_from=gitee_search#https://gitee.com/erzhongxmu/JeeWMSapp-uni

The "Inventory" and "In-library Management" functions of JEEWMS are good, it is recommended to refer to

The framework of JEEWMS is too old and is not easy to maintain.

**Comment by Singosgu** [comment]
tks
